src/watcher/watcher.worker.ts

```diff
@@ -399,15 +399,15 @@ class Parser {
             let datebits = element.dataset.date.split(/(?<!^)-/);
             const date = {
                 year: parseInt(datebits[0]),
-                month: parseInt(datebits[1]),
+                month: parseInt(datebits[1])-1,
```

Comment:
What problem is this solving? Can you give me a working example of this not working correctly?

Reply:
Hello, I just started using Fantasy Calendar and Timelines for TTRPG notes and am running into the issue this is meant to solve. I'm glad I looked at open PRs before I tried to track it down.

I'm using the built-in Golarion calendar, where Calistril is the second month. Our adventure is beginning on 23 Calistril 4710, so I add a timeline event with a `data-date` attribute of `4710-02-23`. After rescanning, the event appears on my calendar on 23 Pharast 4710 (one month late). I'm guessing this is due to 0 indexing somewhere and the `-1` here, I'm assuming, is to correct for that.

![image](https://user-images.githubusercontent.com/40412287/215369756-4432ebc4-6af7-4f19-8012-d1c824ff6cbf.png)

![image](https://user-images.githubusercontent.com/40412287/215369825-d15b0df6-a4f9-40ee-ac2c-7d55ddb8d3af.png)

![image](https://user-images.githubusercontent.com/40412287/215369861-e8589a28-09fc-4797-834e-1750f938a650.png)